### PR TITLE
Fix lint and various other cosmetic changes

### PR DESCRIPTION
**Pull request by sohkai**

### CODE REVIEW

File: src/components/TransactionProgress/TransactionProgress.js [hunk 1]
```diff
@@ -9,18 +9,24 @@ import {
...
-  font,
   theme,
 } from '../..'
+import { noop } from '../../utils'
 
 class TransactionProgress extends React.Component {
   static propTypes = {
+    openerRef: PropTypes.instanceOf(Element).isRequired,
+    top: PropTypes.string,
+    left: PropTypes.string,
```
Comment (sohkai) [inline src/components/TransactionProgress/TransactionProgress.js]: This really seems like an abstraction leak on the `Popover`, but not sure how best to handle this other than just passing all other props down to the `Popover` or having a dedicated `popover` prop :/

Comment (bpierre) [reply src/components/TransactionProgress/TransactionProgress.js]: Yeah I think we should just pass `openerRef` here. I would also like to move `TransactionProgress` to `TransactionProgressPopover`, and `TransactionProgress` would be a component that renders a badge opening `TransactionProgressPopover`. That way, people wouldn’t have to deal with `openerRef` at all.

File: src/providers/Root/Root.js [hunk 2]
```diff
@@ -1,8 +1,12 @@
 import React from 'react'
+import PropTypes from 'prop-types'
 
 const { Provider, Consumer } = React.createContext(null)
 
 class RootProvider extends React.Component {
+  static propTypes = {
+    children: PropTypes.node,
```
Comment (AquiGorka) [inline src/providers/Root/Root.js]: Shouldn't `children` be of type `func`?

Comment (bpierre) [reply src/providers/Root/Root.js]: For the consumer yes, but this is the provider :-)

Comment (AquiGorka) [reply src/providers/Root/Root.js]: Ahhh, thank you very much for calrifying. :) TIL.

**Pull request merged**